### ISSUE TITLE
Playtest Answers - Martin (J)

### ISSUE THREAD

**Issue by marafrass** [FullSiliconAlchemist/LevelDesign_UnityIntro#1]
I figured these were the playtest questions - let me know if it was just general game design questions and I'll replay/update!
EDIT: Double-checked and saw I had an outdated version - updated the files and redid the questions!

_Is the game and objective clear?_
Definitely - I think the on-screen UI was really clear and concise, but more than that I think the bullseye icon below the ball in/decreasing in size as it bounces made it really clear.

_Is the game easy to play/understand?_
Refer to answer #1 - very simple, very easy. You understand what you need to do in seconds. Basic WASD controls feel intuitive. A little unsure what the boundaries were around the map, it's hard to always picture where the invisible walls will be.

_Does a simple game mechanic still provide an entertainment factor?_
Most likely, but it would probably benefit from the original idea of two players! Either that or powerups, or something that makes the game more difficult as you play.

_What is the overall difficulty level given to this game without any power-ups or game-intiated help?_
Very easy so far, but I feel like there's opportunities for development or adding on ideas. Kind of answered this in the answer above this!

**Comment by FullSiliconAlchemist** [comment]
Sup Martin, I think you played the game before I had the chance to update
it! Your input is valid for the older version but if you have time download
the latest version and play it again. I've implemented a score system and I
think I specified the game's mechanics a bit more at the start of the game
with some UI.

Thanks!

**Comment by marafrass** [comment]
Yeah you're totally right! Sorry bout that - I downloaded the build at the end of last class without checking for updates when I tried it today. I updated it the notes just now after playing the real thing!

**Comment by FullSiliconAlchemist** [comment]
Awesome, thanks for taking the extra time!
